src/__percy__/panels.percy.js

```diff
@@ -0,0 +1,278 @@
+import React from 'react';
+import {TestEditor, fixtures, plotly} from 'lib/test-utils';
+
+import {PanelMenuWrapper} from '../components';
+
+import GraphCreatePanel from '../default_panels/GraphCreatePanel';
+import GraphTransformsPanel from '../default_panels/GraphTransformsPanel';
+import StyleTracesPanel from '../default_panels/StyleTracesPanel';
+import StyleLayoutPanel from '../default_panels/StyleLayoutPanel';
+import StyleNotesPanel from '../default_panels/StyleNotesPanel';
+import StyleAxesPanel from '../default_panels/StyleAxesPanel';
+import StyleLegendPanel from '../default_panels/StyleLegendPanel';
+import StyleColorbarsPanel from '../default_panels/StyleColorbarsPanel';
+import StyleShapesPanel from '../default_panels/StyleShapesPanel';
+import StyleImagesPanel from '../default_panels/StyleImagesPanel';
+import StyleSlidersPanel from '../default_panels/StyleSlidersPanel';
+import StyleUpdateMenusPanel from '../default_panels/StyleUpdateMenusPanel';
+
+import '../../dev/styles.css';
+import '../styles/main.scss';
+import './percy.css';
+
+const panelFixture = (Panel, group, name, data = {}) => {
+  return (
+    <div className="plotly_editor">
+      <TestEditor plotly={plotly} {...fixtures.scatter(data)}>
+        <PanelMenuWrapper>
+          <Panel group={group} name={name} />
+        </PanelMenuWrapper>
+      </TestEditor>
+    </div>
+  );
+};
+
+/* eslint-disable no-magic-numbers */
+
+percySnapshot('GraphCreatePanel2', () =>
```

Comment:
can you remove the `2` here and delete all the old snapshots from Percy as well please so we don't get `GraphCreatePanel3` and `Radio` and such?

Reply:
if you can't delete stuff that's ok, I guess it'll just be 'unchanged' forever

Reply:
Unless we do the whole "make new project, disable old one", we'll be stuck not only with unchanged forever, but also with GraphCreatePanel3, because GraphCreatePanel is already taken and it doesn't let me overwrite it for some reason - Percy bug, they're now aware of it, and will probably fix it eventually.